service Greeter {
  // Sends a greeting
  rpc getName (NameRequest) returns (NameReply) {}
}

// The request message containing the user's name.
message NameRequest {
  string id = 1;
}

// The response message containing the greetings
message NameReply {
    int32 id = 1;
  string nconst = 2;
  string primaryName = 3;
  string birthYear = 4; 
  string deathYear = 5;
  string primaryProfession = 6;
  string knownForTitles = 7;
}
